### PR TITLE
Add firefox_nss crashreport option and code refactoring in DRY

### PR DESCRIPTION
**Pull request by jouyingbin**
**Description:**
Attempt to get the effective log from firefox while crash. Adding the crash reporter option will be a way to resolve the gap. Currently it is unable to reproduce the crash in the latest build, but it is worth adding the enhancement for firefox.

1. Add firefox_nss crashreport option function for getting the useful crash log
2. Add some sub function for firefox preferences testing and code refactoring in DRY

- Related ticket: https://progress.opensuse.org/issues/80754
- Needles: NA
- Verification run: https://openqa.suse.de/tests/6110245

### CODE REVIEW

File: tests/fips/mozilla_nss/firefox_nss.pm [hunk 1]
```diff
@@ -29,6 +29,37 @@ sub quit_firefox {
...
+    my $firefox_crash_dir = '/home/bernhard/.mozilla/firefox/Crash\ Reports';
+    my $crash_report      = '/home/bernhard/.mozilla/firefox/Crash\ Reports/crashreporter.ini';
+
+    assert_script_run("ls $firefox_crash_dir");
+
+    assert_script_run("sed -i -e 's/EmailMe=0/EmailMe=1/g' $crash_report");
+    assert_script_run("sed -i -e 's/SubmitReport=0/SubmitReport=1/g' $crash_report");
+    assert_script_run "cat $crash_report";
+    assert_script_run "cat $crash_report | grep 'EmailMe=1'";
+    assert_script_run "cat $crash_report | grep 'SubmitReport=1'";
```
Comment (rfan1) [inline tests/fips/mozilla_nss/firefox_nss.pm:45]: Ben, can you please use the same code style for assert_script_run()?

Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm:45]: SURE, you are definitely right :)

Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm]: Resolved.

**Review by lilyeyes — APPROVED** [review]
LGTM.

**Review by rfan1 — APPROVED** [review]
LGTM

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**